### PR TITLE
Built version for npm

### PR DESCRIPTION
**Pull request by weikinhuang**

### CODE REVIEW

File: dist/jasmine-stray-timers.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
+/******/ (function(modules) { // webpackBootstrap
```
Comment (weikinhuang) [inline dist/jasmine-stray-timers.js:1]: this file can be ignored, it's just the output of npm run webpack

**Comment by matt-oconnell** [comment]
:+1:

**Review by kaicataldo — APPROVED** [review]
LGTM

**Comment by kaicataldo** [comment]
Actually, do we want to not commit the dist bundle to GitHub?

**Comment by kaicataldo** [comment]
For instance, with Focss, we only commit the src files to GitHub, and only publish the lib files to npm

**Pull request merged**